src/neo-vm/OpCode.cs

```diff
@@ -233,6 +233,9 @@ public enum OpCode : byte
         /// Pop the top value of the stack, if it false, then exit vm execution and set vm state to FAULT.
         /// </summary>
         ASSERT = 0x38,
+        /// <summary>
+        /// It turns the vm state to FAULT immediately, but it can be caught.
```

Comment:
This comment is incorrect.

Reply:
I got it with this comment https://github.com/neo-project/neo-vm/pull/300#discussion_r394149485

Reply:
But https://github.com/neo-project/neo-vm/pull/300#discussion_r394149485 is talking about `ABORT`, not `THROW`.